### ISSUE TITLE
Split Screen UI

### ISSUE THREAD

**Issue by thehabes** [CenterForDigitalHumanities/TPEN-NL#130]
In the French interface, some have header and/or footer and/or site nav, some don't.  Look at 'French Script and Hands', then 'Historical Maps'.  You will see it.

**Comment by thehabes** [comment]
**In French...**
Historical Maps and Calligraphy Books show the header in the split screen content, none of the others do
![image](https://user-images.githubusercontent.com/3287006/176249876-2399b3ae-932d-4bf0-bec9-dec3c793cce6.png)
Partial Transcriptions and Background Essays are missing

**In Italian...**
Abbreviations has no content.
Symbols has no content.
Teaching has no content.
In French when there is no content, there is a special message in the split screen area.  There is a different one in Italian.  We should make them the same.

**Comment by thehabes** [comment]
Fixed the header issues, all split screens are consistent.

As part of this, we also noticed the class .markdown-body was making the split screens overflow just a little, causing a horizontal scroll when there was no need for one.  `box-sizing:border-box` was added to the shared CSS to tighten it up.

The only thing left are the known 'no content' split screens, which are their own ticket.  We can close this one.
